lite/Dockerfile

```diff
@@ -3,6 +3,13 @@ ENV LANG C.UTF-8
 
 RUN apk update
 RUN apk upgrade
+
+# Adding LLVM-15 APT repository and installing it
+# LLVM-15 is only present starting from the v3.17 Alpine version
+RUN apk add --repository=http://dl-cdn.alpinelinux.org/alpine/v3.17/main llvm15
```

Comment:
Why do we need llvm-15 in the lite and not just in the full image ?

Reply:
Adding it after this line will ends up in a conflict with I don't know what making it unusable.
This is probably because we are adding a package not meant for this alpine version but as of now that's the best version we found with @lpascal-ledger and @apaillier-ledger  :/

Reply:
Interesting :thinking: